content/applications/sales/subscriptions/renewals.rst

```diff
@@ -81,6 +87,36 @@ also appears at the top of the sales order.
 When clicked, Odoo reveals an :guilabel:`MRR Analysis` page, detailing the monthly recurring revenue
 related to this specific subscription.
 
+.. important::
+   On rare occasions, automatic payment can fail and result in a *Payment Failure* flag if there is
+   an error in the payment method.
+
+   This is done to prevent the system from charging the customer again the next time a schedule
+   action is run. Because the status of the payment is unknown, Odoo requests a manual operation to
+   check if the payment has been made, before the payment can be used again.
+
+   To do this, navigate to :menuselection:`Subscriptions app --> Subscriptions --> Quotations`.
+   Click into the desired subscription, then check the Chatter to see if the payment was made.
+
+   If the payment **was not** made, first enter :doc:`debug mode
+   <../../general/developer_mode>`. Then, click the :guilabel:`Other Info` tab, and untick the
```

Comment:
I would double check this section (try running make review again) I think there is an early line break on line 101.